integration/hive/src/main/java/org/apache/carbondata/hive/MapredCarbonInputFormat.java

```diff
@@ -202,7 +212,7 @@ protected void setFilterPredicates(Configuration configuration, CarbonTable carb
   }
 
   private QueryModel getQueryModel(Configuration configuration, String path)
-      throws IOException, InvalidConfigurationException {
+      throws IOException, InvalidConfigurationException, SQLException {
```

Comment:
Which method is throwing SQLException?

Reply:
getTableInfo() is throwing SQLException which is called from getCarbonTable().